wire-protoc-integration/src/main/java/com/squareup/wire/protocwire/Plugin.kt

```diff
@@ -0,0 +1,260 @@
+package com.squareup.wire.protocwire
+
+import com.google.protobuf.ByteString
+import com.google.protobuf.CodedOutputStream
+import com.google.protobuf.DescriptorProtos
+import com.google.protobuf.Descriptors
+import com.google.protobuf.DynamicMessage
+import com.google.protobuf.ExtensionRegistry
+import com.google.protobuf.compiler.PluginProtos
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+
+/**
+ * Front-end for protoc code generator plugins written in Kotlin (translated from Java).
```

Comment:
From which Java class has it been translated? Can we link it here?

Reply:
Added!